packages/biometric-ed25519/src/index.ts

```diff
@@ -129,3 +139,13 @@ export const getKeys = async (username: string): Promise<[KeyPair, KeyPair]> =>
 export const isPassKeyAvailable = async (): Promise<boolean> => {
     return window.PublicKeyCredential?.isUserVerifyingPlatformAuthenticatorAvailable?.() || false;
 };
+
+// To check if current device is supported
+export const isDeviceSupported = async (): Promise<boolean> => {
+    try {
+        await createKey('test-device');
```

Comment:
Is there any way to check for support without side effects?

I see that the current implementation uses the response from `createKey`, so it's probably not a straightforward thing. But if you are aware of a less intrusive check it would be helpful to document in a comment or follow-up issue.

If this is the necessary approach however, would it be better to use a different key each time? I.e. is the outcome the same if the `test-device` key doesn't exist vs was previously created?

Reply:
After spending some time researching how to do this we found that the only way to do it was by using the response from `createKey`. The outcome should be the same after each call.

Reply:
Thank you for the clarification